<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="Evolutionary Signatures of Intrinsically Disordered Regions">
	<title>Evolutionary Signature Display</title>
    <link rel="stylesheet" href="{{ url_for('static', filename='css/styles.css') }}">
    <style>
        table, th, td {
        border: 1px solid black;
        border-collapse: collapse;
        padding: 5px
        }
        tr:hover {background-color: #f5f5f5;}
        thead{
            background-color: #dddddd;
    }
    </style>
</head>
<body>
	<header>
		<div class="container">
		<a href="{{ url_for('index') }}"><h1>Evolutionary Signatures <span class="shine">of Intrinsically Disordered Regions</span></h1></a>
		</div>
	</header>
	<div class="topnav">
		<div class=container> 
			<a href="{{ url_for('index') }}">home</a>
			<a href="{{ url_for('about') }}">About</a>
			<a href="{{ url_for('contact') }}">Contact</a>
			</div>
	    </div>

	<div class="main">
		<div class="container">
			<div class="column sidenav">
				<form action="{{ url_for('index') }}">
				<button class="button grad">SEARCH</button>
				</form>
				<form action="{{ url_for('cluster') }}">
				<button class="button grad">CLUSTER</a></button>
				</form>
				<form action="{{ url_for('faq') }}">
				<button class="button grad">FAQ</button>
				</form>
				<form action="{{ url_for('download') }}">
				<button class="button grad">DOWNLOAD</button>
				</form>
            </div>
            
    <div class="column right " style="padding:5px">
        <h2>Molecular Features Table</h2>
        <table class="table">
            <thead><tr><th title="Field #1">ID</th>
            <th title="Field #2">Name</th>
            <th title="Field #3">Regular expression (regex)</th>
            <th title="Field #4">Type</th>
            <th title="Field #5">Source</th>
            <th title="Field #6">Description</th>
            <th title="Field #7">Reference</th>
            </tr></thead>
            <tbody><tr>
            <td>AA_S</td>
            <td>S content</td>
            <td>S</td>
            <td>Amino acid content</td>
            <td>NA</td>
            <td>Fraction of S residues</td>
            <td>(Haynes et al., 2006)</td>
            </tr>
            <tr>
            <td>AA_P</td>
            <td>P content</td>
            <td>P</td>
            <td>Amino acid content</td>
            <td>NA</td>
            <td>Fraction of P residues</td>
            <td>(Marsh and Forman-Kay, 2010; Neduva and Russell, 2005; Simon and Hancock, 2009)</td>
            </tr>
            <tr>
            <td>AA_T</td>
            <td>T content</td>
            <td>T</td>
            <td>Amino acid content</td>
            <td>NA</td>
            <td>Fraction of T residues</td>
            <td>Reviewed in (Van Der Lee et al., 2014)</td>
            </tr>
            <tr>
            <td>AA_A</td>
            <td>A content</td>
            <td>A</td>
            <td>Amino acid content</td>
            <td>NA</td>
            <td>Fraction of A residues</td>
            <td>(Perez et al., 2014)</td>
            </tr>
            <tr>
            <td>AA_H</td>
            <td>H content</td>
            <td>H</td>
            <td>Amino acid content</td>
            <td>NA</td>
            <td>Fraction of H residues</td>
            <td>(Marsh and Forman-Kay, 2010)</td>
            </tr>
            <tr>
            <td>AA_Q</td>
            <td>Q content</td>
            <td>Q</td>
            <td>Amino acid content</td>
            <td>NA</td>
            <td>Fraction of Q residues</td>
            <td>(Alberti et al., 2009; Halfmann et al., 2011)</td>
            </tr>
            <tr>
            <td>AA_N</td>
            <td>N content</td>
            <td>N</td>
            <td>Amino acid content</td>
            <td>NA</td>
            <td>Fraction of N residues</td>
            <td>(Alberti et al., 2009; Halfmann et al., 2011)</td>
            </tr>
            <tr>
            <td>AA_G</td>
            <td>G content</td>
            <td>G</td>
            <td>Amino acid content</td>
            <td>NA</td>
            <td>Fraction of G residues</td>
            <td>(Elbaum-Garfinkle et al., 2015)</td>
            </tr>
            <tr>
            <td>kappa</td>
            <td>Kappa</td>
            <td>NA</td>
            <td>Charge properties</td>
            <td>localCIDER</td>
            <td>Measure of separation between positively versus negatively charged residues</td>
            <td>(Das and Pappu, 2013; Holehouse et al., 2017)</td>
            </tr>
            <tr>
            <td>omega</td>
            <td>Omega</td>
            <td>NA</td>
            <td>Charge properties</td>
            <td>localCIDER</td>
            <td>Measure of separation between charged residues and prolines versus all other residues</td>
            <td>(Holehouse et al., 2017; Martin et al., 2016)</td>
            </tr>
            <tr>
            <td>FCR</td>
            <td>Fraction of charged residues</td>
            <td>NA</td>
            <td>Charge properties</td>
            <td>localCIDER</td>
            <td>FCR: basic fraction + acidic fraction</td>
            <td>(Holehouse et al., 2017; Mao et al., 2013)</td>
            </tr>
            <tr>
            <td>NCPR</td>
            <td>Net charge per residue </td>
            <td>NA</td>
            <td>Charge properties</td>
            <td>localCIDER</td>
            <td>NCPR: basic fraction - acidic fraction</td>
            <td>(Holehouse et al., 2017; Mao et al., 2013, 2010)</td>
            </tr>
            <tr>
            <td>net_charge</td>
            <td>net charge</td>
            <td>NA</td>
            <td>Charge properties</td>
            <td>Literature /localCIDER</td>
            <td>Net charge (# [RK] - # [DE])</td>
            <td>(Daughdrill et al., 2007; Strickfaden et al., 2007; Zarin et al., 2017)</td>
            </tr>
            <tr>
            <td>net_charge_P</td>
            <td>net charge with phosphorylation of [ST]P consensus sites</td>
            <td>NA</td>
            <td>Charge properties</td>
            <td>Literature</td>
            <td>Net charge as influenced by phosphorylation of consensus sites</td>
            <td>(Strickfaden et al., 2007; Zarin et al., 2017)</td>
            </tr>
            <tr>
            <td>SCD</td>
            <td>Sequence charge decoration</td>
            <td>NA</td>
            <td>Charge properties</td>
            <td>Literature</td>
            <td>Measure of separation between positively versus negatively charged residues</td>
            <td>(Sawle and Ghosh, 2015)</td>
            </tr>
            <tr>
            <td>RK_ratio</td>
            <td>R/K ratio</td>
            <td>NA</td>
            <td>Charge properties</td>
            <td>Literature</td>
            <td>Ratio of arginine to lysine residues (#R + 1) /(#K + 1)</td>
            <td>(Vernon et al., 2018)</td>
            </tr>
            <tr>
            <td>ED_ratio</td>
            <td>E/D ratio</td>
            <td>NA</td>
            <td>Charge properties</td>
            <td>NA</td>
            <td>Ratio of glutamic acid to aspartic acid residues (#E + 1)/(#D + 1)</td>
            <td> NA</td>
            </tr>
            <tr>
            <td>CLV_Separin_Fungi</td>
            <td>Separase cleavage motif</td>
            <td>S[IVLMH]E[IVPFMLYAQR]GR.</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Separase cleavage site, best known in sister chromatid separation. Also involved in stabilizing the anaphase spindle and centriole disengagement.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>DEG_APCC_KENBOX_2</td>
            <td>APCC-binding Destruction motif</td>
            <td>.KEN.</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Motif conserving the exact sequence KEN that binds to the APC/C subunit Cdh1 causing the protein to be targeted for  26S proteasome mediated degradation.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>DEG_APCC_TPR_1</td>
            <td>APCC_TPR-docking motif</td>
            <td>.[ILM]R</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>This short C-terminal motif is present in co-activators, the Doc1/APC10 subunit and some substrates of the APC/C and mediates direct binding to TPR-containing APC/C core subunits.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>DOC_CKS1_1</td>
            <td>Cks1 ligand</td>
            <td>[MPVLIFWYQ].(T)P..</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Phospho-dependent motif that mediates docking of CDK substrates and regulators to cyclin-CDK-bound Cks1.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>DOC_MAPK_DCC_7</td>
            <td>MAPK docking motif</td>
            <td>[RK].{2,4}[LIVP]P.[LIV].[LIVMF]|[RK].{2,4}[LIVP].P[LIV].[LIVMF]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>A kinase docking motif mediating interaction towards the ERK1/2 and p38 subfamilies of MAP kinases</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>DOC_MAPK_gen_1</td>
            <td>MAPK docking motif</td>
            <td>[KR]{0,2}[KR].{0,2}[KR].{2,4}[ILVM].[ILVF]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>MAPK interacting molecules (e.g. MAPKKs, substrates, phosphatases) carry docking Motifs that help to regulate specific interaction in the MAPK cascade. The classic Motifs approximates (R/K)xxxx#x# where # is a hydrophobic residue.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>DOC_MAPK_HePTP_8</td>
            <td>MAPK docking motif</td>
            <td>([LIV][^P][^P][RK]....[LIVMP].[LIV].[LIVMF])|([LIV][^P][^P][RK][RK]G.{4,7}[LIVMP].[LIV].[LIVMF])</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>A kinase docking motif that interacts with the ERK1/2 and p38 subfamilies of MAP kinases.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>DOC_PP1_RVXF_1</td>
            <td>PP1-docking motif RVXF</td>
            <td>..[RK].{0,1}[VIL][^P][FW].</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Protein phosphatase 1 catalytic subunit (PP1c) interacting Motifs binds targeting proteins that dock to the substrate for dephosphorylation. The motif defined is [RK]{0,1}[VI][^P][FW].</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>DOC_PP2B_PxIxI_1</td>
            <td>Calcineurin (PP2B)-docking motif PxIxI</td>
            <td>.P[^P]I[^P][IV][^P]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Calcineurin substrate docking site, leads to the effective dephosphorylation of serine/threonine phosphorylation sites.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>LIG_APCC_Cbox_2</td>
            <td>APC/C_Apc2-docking motif</td>
            <td>DR[YFH][ILFVM][PA]..</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Motifs in APC/C co-activators that mediates binding to the APC/C core, possibly the catalytic Apc2 subunit. This second variant defines the motif in APC/C co-activators from TAXON:4751 and TAXON:554915.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>LIG_AP_GAE_1</td>
            <td>Gamma-adaptin ear interaction motif</td>
            <td>[DE][DES][DEGAS]F[SGAD][DEAP][LVIMFD]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>The acidic Phe motif mediates the interaction between a set of accessory proteins and the gamma-ear domain (GAE) of GGAs and AP-1. Proposed roles: in clathrin localization and assembly on TGN/endosome membranes and in traffic between the TGN and endosome.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>LIG_CaM_IQ_9</td>
            <td>Helical calmodulin binding motif</td>
            <td>[ACLIVTM][^P][^P][ILVMFCT]Q[^P][^P][^P][RK][^P]{4,5}[RKQ][^P][^P]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Helical peptide motif responsible for Ca2+-independent binding of the CaM . The motif is manly characterized by a hydrophobic residue at position 1, a highly conserved Gln at position 2, basic charges at positions 6 and 11, and a variable Gly at position 7</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>LIG_EH_1</td>
            <td>EH ligand</td>
            <td>.NPF.</td>
            <td>Motifs</td>
            <td>ELM/PhyloHMM</td>
            <td>NPF motif interacting with EH domains, usually during regulation of endocytotic processes</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>LIG_eIF4E_1</td>
            <td>eIF4E binding motif</td>
            <td>Y....L[VILMF]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Motif binding to the dorsal surface of eIF4E.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>LIG_GLEBS_BUB3_1</td>
            <td>GLEBS motif</td>
            <td>[EN][FYLW][NSQ].EE[ILMVF][^P][LIVMFA]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Gle2-binding-sequence motif</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>LIG_LIR_Gen_1</td>
            <td>Atg8 protein family ligands</td>
            <td>[EDST].{0,2}[WFY]..[ILV]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Canonical LIR motif that binds to Atg8 protein family members to mediate processes involved in autophagy.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>LIG_PCNA_PIPBox_1</td>
            <td>PCNA binding PIP box</td>
            <td>((^.{0,3})|(Q)).[^FHWY][ILM][^P][^FHILVWYP][HFM][FMY]..</td>
            <td>Motifs</td>
            <td>ELM/PhyloHMM</td>
            <td>The PCNA binding PIP box  motif is found in proteins involved in DNA replication, repair and cell cycle control.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>LIG_SUMO_SIM_par_1</td>
            <td>SUMO interaction site</td>
            <td>[DEST]{0,5}.[VILPTM][VIL][DESTVILMA][VIL].{0,1}[DEST]{1,10}</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Motif for the parallel beta augmentation mode of non-covalent binding to SUMO protein.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>MOD_CDK_SPxK_1</td>
            <td>CDK Phosphorylation Site</td>
            <td>...([ST])P.[KR]</td>
            <td>Motifs</td>
            <td>ELM/Condens</td>
            <td>Canonical version of the CDK phosphorylation site which shows specificity towards a lysine/arginine residue at the [ST]+3 position.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>MOD_LATS_1</td>
            <td>LATS kinase phosphorylation motif</td>
            <td>H.[KR]..([ST])[^P]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>The LATS phosphorylation motif is recognised by the LATS kinases for Ser/Thr phosphorylation. Substrates are often found toward the end of the Hippo signalling pathway.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>MOD_SUMO_for_1</td>
            <td>Sumoylation site</td>
            <td>[VILMAFP](K).E</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Motif recognised for modification by SUMO-1</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>TRG_ER_FFAT_1</td>
            <td>FFAT motif</td>
            <td>[DE].{0,4}E[FY][FYK]D[AC].[ESTD]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>VAP-A/Scs2 MSP-domain binding FFAT (diphenylalanine [FF] in an Acidic Tract) motif</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>TRG_Golgi_diPhe_1</td>
            <td>ER export signals</td>
            <td>Q.{6,6}FF.{6,7}</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>ER to Golgi anterograde transport signal found at the C-terminus of type I ER-CGN integral membrane cargo receptors (cytoplasmic in this topology), it binds to COPII.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>TRG_NLS_MonoExtN_4</td>
            <td>NLS classical Nuclear Localization Signals</td>
            <td>(([PKR].{0,1}[^DE])|([PKR]))((K[RK])|(RK))(([^DE][KR])|([KR][^DE]))[^DE]</td>
            <td>Motifs</td>
            <td>ELM</td>
            <td>Monopartite variant of the classical basically charged NLS. N-extended version.</td>
            <td>(Dinkel et al., 2016) </td>
            </tr>
            <tr>
            <td>MOD_CDK_STP</td>
            <td>CDK phosphorylation motif</td>
            <td>[ST]P</td>
            <td>Motifs</td>
            <td>Condens</td>
            <td>NA</td>
            <td>(Holt et al., 2009; A. C. W. Lai et al., 2012) </td>
            </tr>
            <tr>
            <td>MOD_MEC1</td>
            <td>Mec1 phosphorylation motif</td>
            <td>[ST]Q</td>
            <td>Motifs</td>
            <td>Condens</td>
            <td>NA</td>
            <td> (A. C. W. Lai et al., 2012; Schwartz et al., 2002)</td>
            </tr>
            <tr>
            <td>MOD_PRK1</td>
            <td>Prk1 phosphorylation motif</td>
            <td>[LIVM]….TG</td>
            <td>Motifs</td>
            <td>Condens</td>
            <td>NA</td>
            <td> (Huang et al., 2003; A. C. W. Lai et al., 2012)</td>
            </tr>
            <tr>
            <td>MOD_IPL1</td>
            <td>Ipl1 phosphorylation motif</td>
            <td>[RK].[ST][LIV]</td>
            <td>Motifs</td>
            <td>Condens</td>
            <td>NA</td>
            <td> (Cheeseman et al., 2002; A. C. W. Lai et al., 2012)</td>
            </tr>
            <tr>
            <td>MOD_PKA</td>
            <td>Pka phosphorylation motif</td>
            <td>R[RK].S</td>
            <td>Motifs</td>
            <td>Condens</td>
            <td>NA</td>
            <td> (Budovskaya et al., 2005; Kemp and Pearson, 1990; A. C. W. Lai et al., 2012; Townsend et al., 1996)</td>
            </tr>
            <tr>
            <td>MOD_CKII</td>
            <td>Ckii phosphorylation motif</td>
            <td>[ST][DE].[DE]</td>
            <td>Motifs</td>
            <td>Condens</td>
            <td>NA</td>
            <td> (A. C. W. Lai et al., 2012; Meggio and Pinna, 2003; Niefind et al., 2007)</td>
            </tr>
            <tr>
            <td>MOD_IME2</td>
            <td>Ime2 phosphorylation motif</td>
            <td>RP.[ST]</td>
            <td>Motifs</td>
            <td>Condens</td>
            <td>NA</td>
            <td> (Holt et al., 2007; J. Lai et al., 2012)</td>
            </tr>
            <tr>
            <td>DOC_PRO</td>
            <td>proline-rich motif</td>
            <td>P..P</td>
            <td>Motifs</td>
            <td>PhyloHMM</td>
            <td>NA</td>
            <td>(Nguyen Ba et al., 2012)</td>
            </tr>
            <tr>
            <td>TRG_ER_HDEL</td>
            <td>ER localization motif</td>
            <td>HDEL</td>
            <td>Motifs</td>
            <td>PhyloHMM</td>
            <td>NA</td>
            <td> (Nguyen Ba et al., 2012)</td>
            </tr>
            <tr>
            <td>TRG_MITOCHONDRIA</td>
            <td>Mitochondrial localization motif</td>
            <td>[MR]L[RK]</td>
            <td>Motifs</td>
            <td>PhyloHMM</td>
            <td>NA</td>
            <td> (Nguyen Ba et al., 2012)</td>
            </tr>
            <tr>
            <td>MOD_ISOMERASE</td>
            <td>Disulfide isomerase motif</td>
            <td>C..C</td>
            <td>Motifs</td>
            <td>PhyloHMM</td>
            <td>NA</td>
            <td> (Nguyen Ba et al., 2012)</td>
            </tr>
            <tr>
            <td>TRG_FG</td>
            <td>FG nucleoporin motif</td>
            <td>F.FG|GLFG</td>
            <td>Motifs</td>
            <td>PhyloHMM</td>
            <td>NA</td>
            <td> (Frey and Görlich, 2009; Nguyen Ba et al., 2012)</td>
            </tr>
            <tr>
            <td>INT_RGG</td>
            <td>RGG motif</td>
            <td>RGG | RG</td>
            <td>Motifs</td>
            <td>Literature</td>
            <td>NA</td>
            <td> (Chong et al., 2018)</td>
            </tr>
            <tr>
            <td>length</td>
            <td>Length</td>
            <td>NA</td>
            <td>Physicochemical properties</td>
            <td>Literature</td>
            <td>Length in log scale</td>
            <td>Reviewed in van der Lee et al. 2014</td>
            </tr>
            <tr>
            <td>acidic</td>
            <td>Acidic residue content</td>
            <td>[DE]</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>NA</td>
            <td>(Warren and Shechter, 2017)</td>
            </tr>
            <tr>
            <td>basic</td>
            <td>Basic residue content</td>
            <td>[RK]</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>NA</td>
            <td>(Fukasawa et al., 2015) </td>
            </tr>
            <tr>
            <td>hydrophobicity</td>
            <td>Hydrophobicity</td>
            <td>NA</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>Kyte-Doolittle scale</td>
            <td>(Kyte and Doolittle, 1982)</td>
            </tr>
            <tr>
            <td>aliphatic</td>
            <td>Aliphatic residue content</td>
            <td>[ALMIV]</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>NA</td>
            <td>(Holehouse et al., 2017)</td>
            </tr>
            <tr>
            <td>polar_fraction</td>
            <td>Polar residue content</td>
            <td>[QNSTGCH]</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>NA</td>
            <td>(Holehouse et al., 2017)</td>
            </tr>
            <tr>
            <td>chain_expanding</td>
            <td>Chain expanding residue content</td>
            <td>[EDRKP]</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>NA</td>
            <td>(Holehouse et al., 2017)</td>
            </tr>
            <tr>
            <td>aromatic</td>
            <td>Aromatic residue content</td>
            <td>[FYW]</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>NA</td>
            <td>(Holehouse et al., 2017)</td>
            </tr>
            <tr>
            <td>disorder_promoting</td>
            <td>Disorder promoting residue content</td>
            <td>[TAGRDHQKSEP]</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>NA</td>
            <td>(Holehouse et al., 2017)</td>
            </tr>
            <tr>
            <td>Iso_point</td>
            <td>Isoelectric point</td>
            <td>NA</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>pH where charge of peptide is neutral</td>
            <td>(Holehouse et al., 2017; Marsh and Forman-Kay, 2010; Tomasso et al., 2016)</td>
            </tr>
            <tr>
            <td>PPII_prop</td>
            <td>PPII propensity</td>
            <td>NA</td>
            <td>Physicochemical properties</td>
            <td>Literature /localCIDER</td>
            <td>Propensity for proline to form left-handed helices</td>
            <td>(Elam et al., 2013; Holehouse et al., 2017)</td>
            </tr>
            <tr>
            <td>REP_Q2</td>
            <td>Q repeat</td>
            <td>Q{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more Q in a row</td>
            <td>(Chavali et al., 2017)</td>
            </tr>
            <tr>
            <td>REP_N2</td>
            <td>N repeat</td>
            <td>N{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more N in a row</td>
            <td>(Chavali et al., 2017)</td>
            </tr>
            <tr>
            <td>REP_S2</td>
            <td>S repeat</td>
            <td>S{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more S in a row</td>
            <td>(Chavali et al., 2017)</td>
            </tr>
            <tr>
            <td>REP_G2</td>
            <td>G repeat</td>
            <td>G{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more G in a row</td>
            <td>(Chavali et al., 2017).</td>
            </tr>
            <tr>
            <td>REP_E2</td>
            <td>E repeat</td>
            <td>E{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more E in a row</td>
            <td>(Chavali et al., 2017)</td>
            </tr>
            <tr>
            <td>REP_D2</td>
            <td>D repeat</td>
            <td>D{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more D in a row</td>
            <td>(Chavali et al., 2017)</td>
            </tr>
            <tr>
            <td>REP_K2</td>
            <td>K repeat</td>
            <td>K{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more K in a row</td>
            <td>(Matsushima et al., 2009; Simon and Hancock, 2009)</td>
            </tr>
            <tr>
            <td>REP_R2</td>
            <td>R repeat</td>
            <td>R{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more R in a row</td>
            <td>(Matsushima et al., 2009; Simon and Hancock, 2009)</td>
            </tr>
            <tr>
            <td>REP_P2</td>
            <td>P repeat</td>
            <td>P{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more P in a row</td>
            <td>(Chavali et al., 2017; Matsushima et al., 2009; Simon and Hancock, 2009)</td>
            </tr>
            <tr>
            <td>REP_QN2</td>
            <td>Q/N repeat</td>
            <td>[QN]{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more Q/N in a row</td>
            <td>(Alberti et al., 2009; Van Der Lee et al., 2014)</td>
            </tr>
            <tr>
            <td>REP_RG2</td>
            <td>R/G repeat</td>
            <td>[RG]{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more R/G in a row; aka &quot;GAR&quot; regions</td>
            <td>(Chong et al., 2018; Matsushima et al., 2009)</td>
            </tr>
            <tr>
            <td>REP_FG2</td>
            <td>F/G repeat</td>
            <td>[FG]{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more F/G in a row</td>
            <td>Reviewed in (Van Der Lee et al., 2014)</td>
            </tr>
            <tr>
            <td>REP_SG2</td>
            <td>S/G repeat</td>
            <td>[SG]{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more S/G in a row</td>
            <td>(Matsushima et al., 2009; Simon and Hancock, 2009)</td>
            </tr>
            <tr>
            <td>REP_SR2</td>
            <td>S/R repeat</td>
            <td>[SR]{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more S/R in a row</td>
            <td>Reviewed in (Van Der Lee et al., 2014)</td>
            </tr>
            <tr>
            <td>REP_KAP2</td>
            <td>K/A/P repeat</td>
            <td>[KAP]{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more K/A/P in a row</td>
            <td>Reviewed in (Van Der Lee et al., 2014)</td>
            </tr>
            <tr>
            <td>REP_PTS2</td>
            <td>P/T/S repeat</td>
            <td>[PTS]{2,}</td>
            <td>Repeats and complexity</td>
            <td>Literature</td>
            <td>Fraction of 2 or more P/T/S in a row</td>
            <td>Reviewed in (Van Der Lee et al., 2014)</td>
            </tr>
            <tr>
            <td>wf_complexity</td>
            <td>Wootton-Federhen sequence complexity</td>
            <td>NA</td>
            <td>Repeats and complexity</td>
            <td>Literature /localCIDER</td>
            <td>Complexity based on SEG algorithm (Wootton and Federhen, 1993), blob length=IDR length, step size = 1</td>
            <td> (Wootton and Federhen, 1993)</td>
            </tr>
            </tbody></table>
    </div>
    </div>
    <div class="footer">
        <p>Alan Moses' Lab © 2019</p>
        <h1b>Links:</h1b><br>
        <a target=blank href="http://www.moseslab.csb.utoronto.ca/">Moses Lab</a>
    </div>
    
</body>

</html>